addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/wonder.xml

```diff
@@ -11,6 +11,16 @@
 			<channel id="color" typeId="color" />
 			<channel id="colorTemperature" typeId="colorTemperature" />
 		</channels>
-	</thing-type>
 
+		<config-description>
+			<parameter-group name="connectionSettings">
```

Comment:
There is no need to define a parameter group if there is a single parameter.

Reply:
Alright, deleted.